### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by DvRkW0lF** [Shadow-Creation-Team/Protector#42] [Feature Request & Ideas]
***Ιδέες***

- **Pass** ↴
Free - Premium 
Κάτι τύπου BattlePass που χει και σε πολλά παιχνίδια. Το pass θα μπορει να χει xp boosts αλλά και `items` ↴

- **__economy system__** ↴      
                                                                                                                                                                                 Economy System όπου θα μπορούμε να ξοδεύουμε και το αντίστοιχο Gold που πέρνουμε στα level-up

- **Site** ↴
Να χουμε ένα Personal Profile στο site και να μπουρούμε να χουμε διαφορετική φοτοProfile - Nickname - Bio κτ.λ

- **Donate** ↴
Εκτός από τους κωδικούς που δίνετε εσεις, να μπορούμε και εμείς οι ίδιοι να `αγοράζουμε` κωδικούς με συγκεκριμένο χρόνο `activation` και συγκεκριμένο αριθμό ατόμων που θα μπορούν να τον χρησιμοποιήσουν και +
**Aγορές του Gold**

- **Profile Command**
Custom Background?

- **Badge**
Badge αναλόγως με το πόσο καίρο ένα άτομο χρησιμοιεί το Bot. Ξέρω είναι δύσκολο στο να εφαρμοστεί αλλά θα ήταν πολλοί ωραίο άμα μπορούσατε να το κάνετε

**Comment by andreaskapsalhs** [comment]
1. Είναι ήδη planned! Πολλά rewards θα έχει και πολλούς **καινούργιους** τρόπους να το κάνεις level up! (Με τα upcoming minigames και άλλα πράγματα που θα μπούν στο bot)

2. Είναι ήδη planned, αλλά είναι κομμάτι μεγαλύτερου συστήματος. 😳 

3. Θα υπάρξει μαζί με το παραπάνω που ανέφερα.

4. Στο Item Shop στην v0.5.2~ μαζί και στο Battlepass θα υπάρχουν ανάλογα XP, Coin, Points Boosters όπου θα είναι items ενός μεγαλύτερου συστήματος. 😳 

5. Θα υπάρχει για purchase και με συγκεκριμένα gold.

6. Πολύ καλή ιδέα! Θα το κοιτάξουμε και μπορεί να γίνει πριν την v0.5!

**Comment by tougrel** [comment]
1, 2 και 5. check issue #24 
3. Θα υπάρχει με το release του 0.5
4. Πολύ καλή ιδέα θα το συζητήσουμε με το team και θα δούμε πως θα το κάνουμε implement.
6. Θα το συζητήσουμε μαζί σας σε κάποιο livestream or στον discord server για το πως μπορούμε να το κάνουμε implement αυτό το part
